jhack/config.py

```diff
@@ -1,15 +1,21 @@
 import os
+import pwd
 from pathlib import Path
 from subprocess import CalledProcessError, check_output
 
 from jhack.logger import logger
 
 IS_SNAPPED = False
 
+USR = pwd.getpwuid(os.getuid())[0]
 
-HOME_DIR = Path("/home") / os.environ["USER"]
-JHACK_DATA_PATH = HOME_DIR / ".config" / "jhack"
-JHACK_CONFIG_PATH = JHACK_DATA_PATH / "config.toml"
+
+if USR == "root":
+    HOME_DIR = "/root"
+else: 
+    HOME_DIR = Path("/home") / os.environ["USER"]
+
+JHACK_CONFIG_PATH = HOME_DIR / ".jhack/config.toml"
```

Comment:
I'd decided to move the jhack config to /home/.config/jhack/ as per the xdg home spec https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html

so this code was correct:
```
JHACK_DATA_PATH = HOME_DIR / ".config" / "jhack"
JHACK_CONFIG_PATH = JHACK_DATA_PATH / "config.toml"
```
if you can revert this change I'm fine with accepting this patch

Reply:
Done